rfcs/0000-rfc-template.md

```diff
@@ -16,11 +16,11 @@ Stage 0: Provide a high level summary of the premise of these changes. Briefly d
 ## Fields
 
 <!--
-Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting.
+Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
```

Comment:
It'll be redundant, but from experience sometimes folks don't always read `rfcs/PROCESS.md` before jumping into the template. So maybe before the first mention of the folder, we reiterate that RFCs making field changes/adds should create an accompanying folder? Maybe even as a separate comment towards the top of the template?